src/Compilers/Core/Portable/SourceGeneration/GeneratorDriver.cs

```diff
@@ -168,6 +176,7 @@ internal GeneratorDriverState RunGeneratorsCore(Compilation compilation, Diagnos
             var stateBuilder = ArrayBuilder<GeneratorState>.GetInstance(state.Generators.Length);
             var constantSourcesBuilder = ArrayBuilder<SyntaxTree>.GetInstance();
             var walkerBuilder = ArrayBuilder<GeneratorSyntaxWalker?>.GetInstance(state.Generators.Length, fillWithValue: null); // we know there is at max 1 per generator
+            var syntaxInputNodes = ArrayBuilder<ISyntaxTransformNode>.GetInstance();
             int receiverCount = 0;
```

Comment:
Consider calling this `walkerCount` because at this point it's only tracking the walkers.